packages/calcite-components/conventions/Documentation.md

```diff
@@ -29,7 +25,7 @@ There are two ways to document deprecations, depending on the API reference. In
 
 ### Usage snippets
```

Comment:
We can remove this section since we [don't have usage snippets anymore](https://github.com/Esri/calcite-design-system/pull/9052) and Lumina doesn't generate READMEs.